astrolabe/utils.py

```diff
@@ -301,15 +302,24 @@ def get_logs(admin_client, project, cluster_name):
     data = admin_client.groups[project.id].logCollectionJobs.post(**params).data
     job_id = data['id']
     
-    while True:
-        LOGGER.debug('Poll job %s' % job_id)
+    timer = Timer()
+    timer.start()
+    ok = False
+    # Wait 10 minutes for logs to get collected - guess as to how long
+    # it might take
+    timeout = 600
+    while timer.elapsed < timeout:
```

Comment:
We should actually subclass the PollerBase class and create an implementation for this but we can leave this be for now.